website/app/GeoKrety/Controller/HealthCheck.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace GeoKrety\Controller;
+
+use GeoKrety\HealthState;
+
+class HealthCheck extends Base {
+    public function get($f3) {
+        $state = new HealthState();
+        $websiteConfig = new \GeoKrety\Service\Config();
+
+        //~ dependencies checks
+        $validationDetails = null;
+        $isConfigOk = $websiteConfig->isValid();
+        if (!$isConfigOk) {
+            $validationDetails = $websiteConfig->validationDetails();
+        }
+
+        $isDbOk = $f3->get('DB') !== false;
+
+        //~ update state
+        $state->setDependencyState('config', $isConfigOk ? HealthState::STATE_OK : HealthState::STATE_KO, $validationDetails);
+        $state->setDependencyState('db', $isDbOk ? HealthState::STATE_OK : HealthState::STATE_KO);
+
+        //~ produce response
```

Comment:
Could this be moved in a `render` function at `HealthState`, like in https://github.com/geokrety/geokrety-website/blob/d33c58691fcfe534b02336d9ded1194c08599237/website/app/GeoKrety/Service/Validation/Waypoint.php#L24-L33

Reply:
sure